pkgs/top-level/python-packages.nix

```diff
@@ -5608,6 +5608,8 @@ in {
 
   safe = callPackage ../development/python-modules/safe { };
 
+  sane = callPackage ../development/python-modules/sane { };
```

Comment:
future proof against a package of the same name being added python-modules
```suggestion
  sane = callPackage ../development/python-modules/sane {
    inherit (pkgs) saneBackends;
  };
```

Reply:
:+1:.